bigbluebutton-html5/imports/ui/components/notifications-bar/container.jsx

```diff
@@ -27,32 +27,26 @@ const STATUS_OFFLINE = 'offline';
 const intlMessages = defineMessages({
   failedMessage: {
     id: 'app.failedMessage',
-    defaultMessage: 'Apologies, trouble connecting to the server.',
     description: 'Message when the client is trying to connect to the server',
   },
   connectingMessage: {
     id: 'app.connectingMessage',
-    defaultMessage: 'Connecting...',
     description: 'Message when the client is trying to connect to the server',
   },
   waitingMessage: {
     id: 'app.waitingMessage',
-    defaultMessage: 'Disconnected. Trying to reconnect in {seconds} seconds...',
     description: 'Message when the client is trying to reconnect to the server',
```

Comment:
If you are removing `defaultMessage` please make the `description`s different - right now they all read `'Message when the client is trying to connect to the server'`